BFS_Connected_Components/Q10_Leetcode_number_of_closed_islands.py

```diff
@@ -0,0 +1,38 @@
+import collections
+def closedIsland(grid):
+    ROWS, COLS = len(grid), len(grid[0])
```

Comment:
All caps is typically reserved for fixed values that never change (e.g. `NUM_SECONDS_IN_DAY = 86400`), but the values of ROWS and COLS can indeed change based on input.

Also, ROWS is not very clear. That could store the data in each of the rows. I'd name this `num_rows`.